sdk/test.py

```diff
@@ -78,10 +78,9 @@ def test_coinbase(self):
         self.assertEqual(str(challenge), challenge_json)
         self.assertEqual(str(solution), solution_json)
 
-        # Skip it because it takes too much time to load the puzzle
```

Comment:
This one is turned on again because CI builds the wheel in release mode so we can run this code fast.